### ISSUE TITLE
Cannot find anything for an SVG generated by Inkscape

### ISSUE THREAD

**Issue by DavidGriffith** [phauer/svg-buddy#1]
Cannot download anything for an SVG generated by Inkscape.  After setting things manually, nothing else would download either.

````
$ svg-buddy-runner.jar cover.svg cover-out.svg
svg-buddy 1.2.2
Detecting Fonts...
Downloading Fonts [Linux Libertine O, Bordeaux Light, BordeauxLight, Animals]...
Downloading font Linux Libertine O and caching it at /home/dave/.svg-buddy/linux-libertine-o.zip...
Font linux-libertine-o could not be found on Google Fonts. Check out https://google-webfonts-helper.herokuapp.com/fonts to find an available font.
Usage: svg-buddy INPUT [OUTPUT] [--optimize]
If the OUTPUT path is not submitted a new file is created with the postfix '-e' in the same directory as the INPUT file. If --optimize is set the postfix '-eo' is used.
usage: gnu
 -o,--optimize   If set, simple optimizations are applied to the output
                 SVG to reduce the file size.
````

**Comment by phauer** [comment]
Can you please provide your `cover.svg` so I can reproduce this issue?

**Comment by DavidGriffith** [comment]
[cover.svg.gz](https://github.com/phauer/svg-buddy/files/5949460/cover.svg.gz)

**Comment by warp9pnt9** [comment]
TL;DR

@DavidGriffith I'd say for now, use another font, or another tool.

@phauer Is it possible to consider adding features to specify fonts locally, or via url?

For the Linux Libertine fonts, see this [Google Fonts issue](https://github.com/google/fonts/issues/12).

Summary:

- The use of the name _Linux_ in _Linux Libertine_ is contentious, as Linux is trademarked, and the font creator used it without permission, so Google can't and won't touch it.
- For Google to touch this font, the font developer and/or Google would have to get express written permission from the holdersf the Linux trademark.
- The Libertine font developer allegedly never responded to inquiries to change the name to Libertine.
- The Libertine font apparently has many problems.
- The Libertine font developer seems to have abandonned the project, or otherwise not maintained it for nearly a decade.
- Someone forked the project to the name Libertinus.
- The new font developer fixed many of the problems and currently maintains the project.
- The Libertinus developer switched the license to the Affero GPL (AGPL).
- Google won't touch any code licensed by the AGPL.  See [this issue](https://github.com/theleagueof/fontship/issues/35).
- The new font developer stubbornly refuses to change the AGPL back to GPL or OFL.
- The new font developer hasn't been able to cite specific valid reasons to justify the decision to use the AGPL, despite being shown specific examples of open source developers being harmed by that choice.
- Although the new developer seems ammenable to the idea of relaxing the AGPL requirement, he seems reluctant to actually follow through, and is waiting for Google to respond to his request to accept the AGPL (good luck with that).

My takeaway is that relying exclusively on Google Fonts is kind of an insane restriction, because these pernicious and intractible problems arise, and are entirely avoidable by placing the responsibility on the developer to choose their own terms and conditions and accept the responsibility and liability of their choices.  The last thing any developer needs is to get embroiled in this kind of quagmire.
